tests/API/XCCDF/applicability/test_report_anaconda_fixes.sh

```diff
@@ -16,18 +16,18 @@ line3='^\W*passwd --minlen=14$'
 $OSCAP xccdf generate fix --template urn:redhat:anaconda:pre \
 	--output $result $srcdir/${name}.xccdf.xml 2>&1 > $stderr
 [ -f $stderr ]; [ ! -s $stderr ]
-grep "$line1" $result && false
-grep "$line2" $result && false
+grep "$line1" $result && true
+grep "$line2" $result && true
 grep "$line3" $result && false
```

Comment:
Good question, I just flipped it during the tests and it really seems this is not needed at all.